tendermint/src/abci/responses.rs

```diff
@@ -59,9 +60,11 @@ pub struct DeliverTx {
     pub info: Info,
 
     /// Amount of gas wanted
+    #[serde(rename = "gasWanted")]
```

Comment:
You can avoid all of these field attribute by putting a `#[serde(rename_all = "camelCase")]` on the struct. See the [section](https://serde.rs/container-attrs.html#rename_all) in the container attribute documentation.

Reply:
I think the camelCase here is a special case, because other places tendermint actually use `_` format.